atlp-rwanda/knights-ecomm-fe
# feat-wishlist-management

What does this PR do?

- This PR enhances the buyer experience by implementing comprehensive management features for the wishlist functionality.

Description of Task Completed

Tasks completed in this PR:

- Designed and integrated UI components for managing the Buyer wishlist.
- Implemented state management using Redux to maintain wishlist state across sessions.
- Added functionality to display wishlist icons on products when users are logged in, enhancing user interaction.
- Ensured seamless integration with backend APIs for wishlist operations, including adding, removing, and updating wishlist items.
- Conducted unit tests to verify functionality and maintain code quality, ensuring robust performance under various scenarios.

How should this be manually tested?

- Ensure You have backend service running and set VITE_APP_API_URL in your .env file.
- Clone this repository and navigate to the project director

`git clone -b ft-register https://github.com/atlp-rwanda/knights-ecomm-fe.git
cd knights-ecomm-fe
git checkout ft-landing-page
npm i
npm run dev`


Screenshots
![Screenshot 2024-07-05 131129](https://github.com/atlp-rwanda/knights-ecomm-fe/assets/142214777/32a64e63-aa50-4fc8-bc2f-e9ec626f5fd1)
![Screenshot 2024-07-05 131226](https://github.com/atlp-rwanda/knights-ecomm-fe/assets/142214777/5ff5ec4b-20d5-4b31-b85c-1bd306cefae4)
![Screenshot 2024-07-05 131258](https://github.com/atlp-rwanda/knights-ecomm-fe/assets/142214777/4d9ad0b0-de86-4e02-8fab-94f117f5f11d)
